action.yml

```diff
@@ -156,16 +156,19 @@ runs:
       echo ::group::Move "'${{ steps.collection-metadata.outputs.fqcn }}'"
       collection to ${{ steps.collection-metadata.outputs.checkout-path }}...
     shell: bash
-  - name: Install ansible-core (${{ inputs.ansible-core-version }})
+  - name: Move collection to test directory
     run: >-
       set -x
       ;
       mkdir -pv
       "${{ steps.collection-metadata.outputs.collection-namespace-path }}"
       ;
-      mv -v
+      rsync -a
```

Comment:
We shouldn't support working in a dirty working dir. I was about to suggest doing the cleanup in the `post` stage (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#post) but it looks like only JS actions support that: https://github.com/actions/runner/issues/1478.

So for now, I'd go for `rm -rf .tmp-ansible-collection-checkout/` as the first step.

Reply:
We would need to `rm -rf "${{ steps.collection-metadata.outputs.collection-namespace-path }}"` as the destination directory of the move is the issue. I can update this PR if that is the route you prefer.

Reply:
@webknjaz change pushed

Reply:
I'd prefer to clean up the whole directory we manage to prevent any similar problems with other dirs. Especially as we never expect external influence on that dir.

Reply:
As a side note: maybe I'll consider using `ln -sf` instead of `mv` in the future. But this is out of the scope here.